### DISCUSSION TITLE
H265: Support HEVC in RTMP/SRT/HLS/WebRTC/DASH/GB28181

### DISCUSSION THREAD

**Discussion by Dvalin21** [roflcoopter/viseron#560] [Ideas]
Can this be added?  

https://github.com/ossrs/srs/issues/465

**Comment by roflcoopter** [comment]
Not really sure what it is you want to have added here, do you want Viseron to include SRS?

**Reply by Dvalin21** [reply]
> Not really sure what it is you want to have added here, do you want Viseron to include SRS?

Yes sir!
